client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -91,6 +136,17 @@ const getTitanMenuItems = ( { email, translate } ) => {
 				comment: 'View the Contacts application for Titan',
 			} ),
 		},
+		{
+			isInternalLink: true,
+			materialIcon: 'delete',
+			onClick: () => {
+				removeTitanMailbox( dispatch, mailbox, onAccountChanged, translate );
+			},
+			key: `remove_mailbox:${ mailbox.mailbox }`,
+			title: translate( 'Remove Mailbox', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Remove mailbox' )` *ES Score: 13*